Jenkinsfile

```diff
@@ -65,48 +90,56 @@ pipeline {
             }
           }
         } // Library Checks and XS2 Tests
-        stage("XS3 Tests and xdoc") {
+        stage("XS3 Build and docs") {
           agent {
             label 'x86_64&&linux'
           }
-          environment {
-            REPO = 'lib_i2s'
-            VIEW = getViewName(REPO)
-            XCORE_AI = 1
-          }
-          options {
-            skipDefaultCheckout()
-          }
           stages {
             stage('Get view') {
               steps {
-                xcorePrepareSandbox("${VIEW}", "${REPO}")
-              }
-            }
-            stage("Build Examples - XS3") {
-              steps {
+                sh 'git clone git@github.com:xmos/test_support'
                 dir("${REPO}") {
-                  xcoreAllAppsBuild('examples')
-                  xcoreAllAppNotesBuild('examples')
+                  checkout scm
+                  installPipfile(false)
                 }
               }
             }
-            stage("Test - XS3") {
+            stage('Run xmosdoc') {
               steps {
-                dir("${REPO}/tests") {
-                  viewEnv {
-                    // reactivating the tools with the newer version
-                    withTools(params.TOOLS_VERSION) {
-                      runPytest()
-                    }
+                dir("${REPO}") {
+                  sh "docker pull ghcr.io/xmos/xmosdoc:$XMOSDOC_VERSION"
```

Comment:
Use buildDocs() from JSL (when available)

Reply:
OK, that's not merged yet so will have to wait